scripts/populate_lambda_ssm_parameters.py

```diff
@@ -82,6 +85,43 @@ def get_admin_user_emails():
     admin_user_emails.append(gcp_service_account_email)
     return ",".join(admin_user_emails)
 
+
+def compare_local():
+    """Compares local env values to deployed values"""
+    ssm_env = get_ssm_lambda_environment()
+    local_env = get_local_lambda_environment()
+    missing_local, diff_local = compare_env_dicts(local_env, ssm_env)
+    if len(missing_local) > 0:
+        print(f"missing values when comparing local env to deployment\n {missing_local}")
+    if len(diff_local) > 0:
+        print(f"Found different values when comparing local env to deployment \n{diff_local}")
+
+
+def compare_stages(stage: str, previous_stage: str = None):
+    """compares lambda env from current deployment stage to the previous stage"""
+    stages = ['dev', 'integration', 'staging', 'prod']
+    compare_stage = None
+    ssm_env_1 = get_ssm_lambda_environment(stage)
+    if previous_stage is not None:
+        compare_stage = previous_stage # comparison stage was defined
+    elif stage in stages[1:]:  # dont check dev.....
+        compare_stage = stages[stages.index(stage) - 1]  # get previous stage in deployment
+    else:
+        print(f"did not find suitable comparison for {stage}, try specifying a previous stage")
+        return
+    ssm_env_2 = get_ssm_lambda_environment(stage=compare_stage)
+    missing, diff = compare_env_dicts(ssm_env_1, ssm_env_2)
+    if len(missing) > 0:
+        print(f"missing values when comparing {stage} to {compare_stage}\n {missing}")
+    if len(diff) > 0:
+        print(f"Found different values when comparing comparing {stage} to {compare_stage}\n{diff}")
+
+
+def compare_env_dicts(d1: dict, d2: dict):
+    missing_env_values = set.symmetric_difference(set(d1.keys()), set(d2.keys()))
+    different_env_values = {k: d2[k] for k in d2 if k in d1 and d2[k] != d1[k]}
```

Comment:
`different_env_values` should probably return both values as a diff for printing a comparison.

Reply:
Noted, will do.